src/components/Actions/Actions.vue

```diff
@@ -720,14 +796,7 @@ export default {
 	 * @return {VNodes} The created VNodes
 	 */
 	render(h) {
-		/**
-		 * Filter the Actions, so that we only get allowed components.
-		 * This also ensure that we don't get 'text' elements, which would
-		 * become problematic later on.
-		 */
-		const actions = (this.$slots.default || []).filter(
-			action => action?.componentOptions?.tag
-		)
+		const actions = getActions(this)
```

Comment:
This should be equivalent to the above function, no? And is easier to understand, I think.
```suggestion
		const actions = (this.$slots.default || this.$scopedSlots.default({ clearFocusTrap: this.clearFocusTrap }) || []).filter(
			action => action?.componentOptions?.tag
		)
```